main.md

```diff
@@ -950,11 +950,11 @@ The Security Considerations of [@RFC6749], [@RFC7662], and [@RFC8414] also apply
 
 # Privacy Considerations {#privacy_considerations}
 
-Implementers must design and use authorization details in a privacy-preserving manner.
+Implementers MUST design and use authorization details in a privacy-preserving manner.
```

Comment:
We're not supposed to have normative requirements in the privacy section.

```suggestion
It is especially important for implementers to design and use authorization details in a privacy-preserving manner.
```

Reply:
There are three other MUST occurrences in Privacy Considerations that should probably be changed too. 

I've never been sure if not supposed to have normative requirements in the privacy section a hard and fast rule. Or recommendation. Or rumor.

Reply:
There's a SHOULD in there now too...

Reply:
I just went ahead and replaced normative keywords with regular language in the privacy considerations 9d57d65